docs/jsdoc/options.jsdoc

```diff
@@ -36,6 +36,10 @@
  * The filename of the template. Required for inclusion and caching unless 
  * you are using {@link module:ejs.renderFile}. Also used for error reporting.
  * 
+ * @property {String}  [root='undefined']
```

Comment:
@TimothyGu Just making sure this is clearly understood; I was asking about quoting `undefined`.

For `filename`, `undefined` is quoted.

However, for `root` you said: 

> `undefined` should not be quoted.

Which is correct; `'undefined'` or `undefined`? Is `filename` erroneously quoting `undefined`?